mobile/native/src/ln_dlc/mod.rs

```diff
@@ -175,7 +177,7 @@ pub fn run(data_dir: String, seed_dir: String) -> Result<()> {
 
         runtime.spawn(async move {
             loop {
-                if let Err(e) = order::handler::check_open_orders() {
+                if let Err(e) = spawn_blocking(order::handler::check_open_orders).await {
```

Comment:
> 🙃 I think it's clearer if we prepend `tokio`, so that one can quickly scan for these calls (or even grep for them).

I considered that for a bit, but then I realised that it's not the same problem as with `sleep` - `spawn_blocking` has *only* a `tokio` version, so it can be grepped easily!